File: javascript/fizz_buzz.js
/* Basic implementation of the fizzbuzz question.
 * Time Complexity - O(n)
 */

var fizzBuzz = function() {

	for (var i = 0; i <= 100; i++) {
		var str = i + ': ';
		if (i % 3 === 0) {
			str +=  "Fizz";
		}
		if (i % 5 === 0) {
			str += "Buzz"
		}
		console.log(str); 
	}
}

fizzBuzz();

File: python/fizz_buzz.py
def fizzbuzz():
	for num in range(1, 101):
		message = ''
		if num % 3 == 0:
			message += 'Fizz'
		if num % 5 == 0:
			message += 'Buzz'
		if not message:
			message = str(num)
		print(message)

fizzbuzz()